crates/node/p2p/src/gossip/driver.rs

```diff
@@ -178,12 +178,12 @@ impl GossipDriver {
 
         match self.swarm.dial(addr.clone()) {
             Ok(_) => {
-                event!(tracing::Level::TRACE, peer=%addr, "Dialed peer");
+                trace!(peer=?addr, "Dialed peer");
```

Comment:
nit, add a target here

```suggestion
                trace!(target: "gossip", peer=?addr, "Dialed peer");
```